lib/index.js

```diff
@@ -872,7 +872,7 @@ KnexMigrator.prototype._rollback = function _rollback(options) {
                 .delete();
         }
 
-        debug('Rollback', task.name);
+        logging.warn('Rollback', task.name);
```

Comment:
Did you mean to change this log line? 😛 I meant the one inside the `migrate` function itself 😁

Reply:
Oooh okay - I went for this one because then it's "global" but yeah I can do it there instead!

Reply:
Yeah reading your comments above - this isn't useful for you at all!